src/sagas/index.js

```diff
@@ -3,12 +3,11 @@ import { takeEvery, call, all, takeLatest, put } from 'redux-saga/effects';
 
 function* createTodoSaga(action) {
   try {
-    let response = yield call(axios.post, 'https://crudcrud.com/api/d830b33c752f4f52b7520389123dbe79/tasks', {name:action.payload}, {
+    let response = yield call(axios.post, 'https://crudcrud.com/api/708f509e6aee4ecb944e7e2c895118cd/tasks', {name:action.payload}, {
       headers: {
         'Content-Type': 'application/json',
       },
     });
-    console.log(response)
     yield call(updateStore)
   } catch (error) {
     console.error(error);
```

Comment:
do you need to pass any payload on which base you are updating store

Reply:
no, it will get all the tasks from the api and store it in the store.